airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java

```diff
@@ -958,55 +1003,56 @@ private UUID getDestinationDefId() throws ApiException {
+    return getDbConfig(destinationPsql, false, true, Type.DESTINATION);
   }
 
   private JsonNode getDestinationDbConfigWithHiddenPassword() {
-    return getDbConfig(destinationPsql, true, true);
+    return getDbConfig(destinationPsql, true, true, Type.DESTINATION);
   }
 
-  private JsonNode getDbConfig(PostgreSQLContainer psql) {
-    return getDbConfig(psql, false, false);
+  private JsonNode getDbConfig(PostgreSQLContainer psql, boolean hiddenPassword, boolean withSchema, Type connectorType) {
+    try {
+      final Map<Object, Object> dbConfig = (IS_KUBE && IS_GKE) ? GKEPostgresConfig.dbConfig(connectorType, hiddenPassword, withSchema)
+          : localConfig(psql, hiddenPassword, withSchema);
+      return Jsons.jsonNode(dbConfig);
+    } catch (Exception e) {
+      throw new RuntimeException(e);
+    }
   }
 
-  private JsonNode getDbConfig(PostgreSQLContainer psql, boolean hiddenPassword, boolean withSchema) {
-    try {
-      final Map<Object, Object> dbConfig = new HashMap<>();
-
-      // don't use psql.getHost() directly since the ip we need differs depending on environment
-      if (IS_KUBE) {
-        if (IS_MINIKUBE) {
-          // used with minikube driver=none instance
-          dbConfig.put("host", Inet4Address.getLocalHost().getHostAddress());
-        } else {
-          // used on a single node with docker driver
-          dbConfig.put("host", "host.docker.internal");
-        }
+  private Map<Object, Object> localConfig(PostgreSQLContainer psql, boolean hiddenPassword, boolean withSchema) throws UnknownHostException {
+    final Map<Object, Object> dbConfig = new HashMap<>();
+    // don't use psql.getHost() directly since the ip we need differs depending on environment
+    if (IS_KUBE) {
+      if (IS_MINIKUBE) {
+        // used with minikube driver=none instance
+        dbConfig.put("host", Inet4Address.getLocalHost().getHostAddress());
       } else {
-        dbConfig.put("host", "localhost");
+        // used on a single node with docker driver
+        dbConfig.put("host", "host.docker.internal");
       }
+    } else {
+      dbConfig.put("host", "localhost");
+    }
 
-      if (hiddenPassword) {
-        dbConfig.put("password", "**********");
-      } else {
-        dbConfig.put("password", psql.getPassword());
-      }
+    if (hiddenPassword) {
+      dbConfig.put("password", "**********");
+    } else {
+      dbConfig.put("password", psql.getPassword());
+    }
 
-      dbConfig.put("port", psql.getFirstMappedPort());
-      dbConfig.put("database", psql.getDatabaseName());
-      dbConfig.put("username", psql.getUsername());
+    dbConfig.put("port", psql.getFirstMappedPort());
+    dbConfig.put("database", psql.getDatabaseName());
+    dbConfig.put("username", psql.getUsername());
 
-      if (withSchema) {
-        dbConfig.put("schema", "public");
-      }
-      return Jsons.jsonNode(dbConfig);
-    } catch (Exception e) {
-      throw new RuntimeException(e);
```

Comment:
curious, why was this removed?

Reply:
@davinchia Oh no this was not removed, I just refactored methods and broke this into two. The new method `localConfig` returns a map and the JSON conversion is done in `getDbConfig`